migration/sql-files/032-add-space-id-wilt.sql

```diff
@@ -1,5 +1,11 @@
 -- Alter the table work_item_link_types
-ALTER TABLE work_item_link_types ADD space_id uuid;
+INSERT INTO spaces (created_at, updated_at, id, name, description) VALUES (now(),now(),'2e0698d8-753e-4cef-bb7c-f027634824a2', 'system.space', 'Initial space');
```

Comment:
With this approach, I wonder how difficult would be to maintain `migration.go` whenever we reach the three digits in our db Version :D. Also, sql.Tx doesn't offer any wide API to avoid passing-hand_typing a whole sql statement :/ (I am disappointed with them).

@kwk I guess that we could do that as a general thing for all the code not only for this PR. That might then be part of another PR.